test/e2e/Makefile

```diff
@@ -28,7 +28,7 @@ export GOPROXY
 REPO_ROOT := $(shell git rev-parse --show-toplevel)
 
 help:  ## Display this help
-	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-25s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
```

Comment:
from @vincepri https://github.com/kubernetes-sigs/cluster-api/pull/3639#discussion_r488747645

>  Was this change intentional? What does it do?

Yes, it simply increases the spaces between the make target and the description in the help so the output is nicer now that there is a new `cluster-templates` target that is longer than the 15spaces used before